config.tutorial.yaml

```diff
@@ -74,6 +74,8 @@ cluster_options:
     exclude_carriers: []
     remove_stubs: true
     remove_stubs_across_borders: true
+    e_threshold_drop_isolated: 10000000
+    e_threshold_merge_isolated: 100000000
```

Comment:
These numbers are a bit ackward
1MW unit would produce 8.76 GWh yearly.
I think that we could safely disregard anything below 5-10 MW -> about 50-100 GWh
The unit shall be added as a comment, and as default values we could use 100GWh?
What do the tests suggest in your cases?

Reply:
Agree, that is a good alternative to a dimensionless threshold :) Changed.

In the case CDR+Zambia the annual loads are of order 1e5..1e7 (which means I suppose MWh, right?) 

For testing purposes I have set 10 and 100 TWh so far. It should be revised further as it feels like the the considered testing case is the one when merging would be a more proper approach as compared with drop

Reply:
Those thresholds are quite high.
Ok for testing but I'd expect lower values, values need checking with the full model later on

Reply:
@davide-f thanks! Let's go for threshold as a share of the overall load? We anyway calculate it to checks

@pz-max great hint :) thanks a lot!

Reply:
> @davide-f thanks! Let's go for threshold as a share of the overall load? We anyway calculate it for checks
> 
> @pz-max great hint :) thanks a lot!

I think having it as a share of the total load may makes things more complicated to understand what happens, I'd keep it as is.
Also because the threshold must be country-based. it cannot be related to the overall demand of all the region of interest.
Moreover, it would make things difficult to compare from big and small countries.
For big countries like china, a 0.1% of the total demand may make disappear even large cities, while for small countries may keep also small buses with few MW of installed capacity.
That is a complication that I would keep for later if needed.

Reply:
> I think having it as a share of the total load may makes things more complicated to understand what happens, I'd keep it as is. Also because the threshold must be country-based. it cannot be related to the overall demand of all the region of interest. Moreover, it would make things difficult to compare from big and small countries. For big countries like china, a 0.1% of the total demand may make disappear even large cities, while for small countries may keep also small buses with few MW of installed capacity. That is a complication that I would keep for later if needed.

Ah, I see. Agree that the absolute value also absolutely make sense. I'll revert it

Reply:
Revised to keep config parameters in a reasonable range. Would it be probably more convenient to use mean power [MW] instead of load [GWh]?

(I'll clean-up the history once we'll complete work! ;) )